README.md

```diff
@@ -32,6 +32,7 @@ operators in production.
 - [Control Flow Primitives](docs/control_flow_primitives.md)
 - [Keeping Reconciliation Loops Short](docs/keeping_reconciliation_loops_short.md)
 - [Using Finalizers](docs/using_finalizers.md)
+- [Troubleshooting Guide](docs/troubleshooting.md)
```

Comment:
That was alphabetical so far.

Reply:
I'll fix this. I was thinking semantics here, but alphabetical ordering works as well.